app/policies/stripe_card_policy.rb

```diff
@@ -10,11 +10,11 @@ def shipping?
   end
 
   def freeze?
-    admin_or_manager? || organizer_and_cardholder?
+    admin_or_manager? || organizer_and_cardholder? || (cardholder? && record.card_grant)
```

Comment:
Just to keep these readable can we make `(cardholder? && record.card_grant)` a method called `grantee?`